src/terrain/clients/iplant_groups/subjects.clj

```diff
@@ -12,10 +12,12 @@
 
 (defn lookup-subject
   "Uses iplant-groups's subject lookup by ID endpoint to retrieve user details."
-  [grouper-user short-username]
-  (:body (http/get (iplant-groups-url "subjects" short-username)
-                   {:query-params {:user grouper-user}
-                    :as           :json})))
+  ([short-username]
+   (lookup-subject (config/grouper-user) short-username))
+  ([grouper-user short-username]
+   (:body (http/get (iplant-groups-url "subjects" short-username)
+                    {:query-params {:user grouper-user}
+                     :as           :json}))))
```

Comment:
The new arity of this function isn't used directly in Terrain at the moment, but having it available in the REPL made development testing much easier.